### PR TITLE
Adds validation to dob on person model

### PR DESCRIPTION
**Pull request by jasoncorum**

### CODE REVIEW

**Comment by jasoncorum** [comment]
Just got back from my first Code for DC event and looking to contribute! This is for the following ticket:

Demographic - DOB Year #152

File: app/models/person.rb [hunk 1]
```diff
@@ -15,7 +15,8 @@ class Person < ActiveRecord::Base
 
   before_validation :initialize_person
   validates_associated :mail_address
-  validates :mail_address, presence: true
+  validates :mail_address, :dob, presence: true
```
Comment (jrunningen) [inline app/models/person.rb]: The `presence: true` means that a person can't be saved to the database unless a birthday has been entered.  That's not right.  We want to allow saving people without birthdays.  Keep in mind that users should be able to leave any part of the form blank and come back to their work later.

**Comment by jasoncorum** [comment]
Cool. Makes sense. Just removed.

Also, I set the dob validation to 120 years ago. Think that is sufficient?

**Comment by jrunningen** [comment]
:cool:

**Pull request merged**